### DISCUSSION TITLE
Student pack not granted

### DISCUSSION THREAD

**Discussion by ciquezada** [community/community#131111] [GitHub Education]
### Hi everyone,

Hello!

My student pack was rejected cause Im currently doing a studentship far from my home country and university. Uploaded the photo with the document that certifies that but it keep beeing rejected

**Comment by paiineli** [comment]
Hello,

I understand the frustration of having your student pack application rejected, especially when you have provided documentation to verify your status. Here are a few suggestions to help you address this issue:

Contact GitHub Support: Reach out directly to GitHub Support to explain your situation. Provide details about your studentship and the documentation you uploaded. They may be able to give you specific feedback on why your application was rejected.

Check Documentation Requirements: Ensure that the document you uploaded meets all of GitHub’s requirements. Sometimes, documents need to be clear, legible, and contain specific information.

Provide Additional Evidence: If possible, upload additional documentation that supports your claim, such as enrollment letters, a student ID, or proof of your studentship.

Clarify Your Situation: In your communication with support, clearly explain your circumstances and how they relate to your student status.

I hope this helps, and that you receive a positive resolution soon!

Best,
